packages/marko-web-theme-monorail/components/gam/define-display-ad.marko

```diff
@@ -6,7 +6,7 @@ import defaultValue from "@parameter1/base-cms-marko-core/utils/default-value";
 
 $ const { GAM, req } = out.global;
 
-$ const adUnit = GAM.getAdUnit({ name: input.name, aliases: input.aliases });
+$ const adUnit = GAM ? GAM.getAdUnit({ name: input.name, aliases: input.aliases }) : {};
 $ const { templateName } = adUnit;
```

Comment:
@Shinsina - if GAM doesn't exist, and templateName doesn't exist, what happens?

Reply:
@brandonbk See production screenshot, it hard fails because if GAM doesn't exist GAM.getAdUnit isn't a valid call as "cannot read property getAdUnit of undefined"